File: examples/python/hello.py
#!/usr/bin/env python3
"""
Sample Python application demonstrating the development environment.
"""

def greet(name: str) -> str:
    """Return a greeting message."""
    return f"Hello, {name}! Welcome to the Devbox environment."

def main():
    """Main entry point."""
    print(greet("Developer"))
    print(f"Python is working correctly!")

if __name__ == "__main__":
    main()

File: examples/typescript/hello.ts
/**
 * Sample TypeScript application demonstrating the development environment.
 */

function greet(name: string): string {
    return `Hello, ${name}! Welcome to the Devbox environment.`;
}

function main(): void {
    console.log(greet("Developer"));
    console.log("TypeScript is working correctly!");
    console.log(`Node.js version: ${process.version}`);
}

main();
